ui.frontend/src/customFunctions.js

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+/**
+ *
+ * @param str {string} json string to convert custom function to object
+ * @return {object} JSON Object after parsing the string as json. In case of
+ * exceptions empty object is returned
+ */
+function toObject(str) {
```

Comment:
these are defined at global level right now. we should put them under scope.

Reply:
That is a limitation of our custom functions right now.

Reply:
But since they are not exported, they would not pollute the window namespace.